http-netty/src/test/groovy/io/micronaut/http/netty/reactive/HandlerPublisherSpec.groovy

```diff
@@ -1,88 +0,0 @@
-package io.micronaut.http.netty.reactive
-
-import io.netty.channel.ChannelDuplexHandler
-import io.netty.channel.ChannelHandlerContext
-import io.netty.channel.embedded.EmbeddedChannel
-import org.reactivestreams.Subscriber
-import org.reactivestreams.Subscription
-import spock.lang.Specification
-
-class HandlerPublisherSpec extends Specification {
```

Comment:
do we test this somewhere else?

Reply:
HandlerPublisher is gone now, the reactive publisher doesn't buffer so it's not susceptible